code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -184,4 +184,9 @@
 	if(istype(bg) && bg.client)
 		return bg
 
-	return src
\ No newline at end of file
+	return src
+
+/mob/living/carbon/human/proc/has_hearing_aid()
+	if(istype(l_ear, /obj/item/device/hearing_aid) || istype(r_ear, /obj/item/device/hearing_aid))
```

Comment:
```suggestion
	if(is_type_in_list(/obj/item/device/hearing_aid, list(l_ear, r_ear)))
```

Idk if you'd consider this more easily readable? Is alright as it is too.

Reply:
naw, i think the current one is fine